Telemetry.md

```diff
@@ -0,0 +1,63 @@
+
+### Telemetry
+
+WrenAI relies on anonymous usage statistics to continuously improve. That's why we ask you to share your usage data with us.
+WrenAI use [Posthog](https://posthog.com/) as out telemetry tool
+
+
+### How Does Telemetry help WrenAI
+
+Based on the telemetry, we can know which features are frequently used and related to our community or in which step an error occured and block our user.
```

Comment:
Based on the telemetry, we can know what features are frequently used and related to our community or in what step an error occurs and blocks our user.